cosmos/dbt/graph.py

```diff
@@ -194,9 +219,12 @@ def load_via_dbt_ls(self) -> None:
                         for line in logfile:
                             logger.debug(line.strip())
 
-                if stderr or "Runtime Error" in stdout:
+                if stderr or "Runtime Error" or "Compilation Error" in stdout:
```

Comment:
It may be better to capture any "Error" (check #488), what do you think?

Reply:
I have changed to just "Error"